### ISSUE TITLE
After update launcher to newest commit (Gradle 7) my packs wont start

### ISSUE THREAD

**Issue by Bartixxx32** [SKCraft/Launcher#465]
Hello, as in topic, i saw there was repo update and i have downloaded it, now as i see my packs wont start, ofc i can revert to piervous builds, but thats not point of that

Version 1.7.10, wont start
1.16.5 seems fine

And what i saw as far, when builded with older version of Modpack creator it started fine

```
To report this error, please provide:

java.lang.NullPointerException
	at com.skcraft.launcher.launch.Runner.addJvmArgs(Runner.java:278)
	at com.skcraft.launcher.launch.Runner.call(Runner.java:143)
	at com.skcraft.launcher.launch.Runner.call(Runner.java:47)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
```
![image](https://user-images.githubusercontent.com/16412925/165412756-f8cae92f-0a7f-4f8c-8ecd-bc1f81764d33.png)

**Comment by hedgehog1029** [comment]
Hah, I did some dependency upgrades but didn't test too far past "it builds" - I will take a look.

The main branch can be a little unstable - I don't really have a proper release system in place. It's been on the TODO list for a while.

**Comment by Bartixxx32** [comment]
sorry, i was wrong, i compared the _upload folder from new build and old, theres no changes except build date, then i guess launcher has issue

**Comment by Bartixxx32** [comment]
Rn, im lost, im gonna try to reproduce it agin, becouse now all works, but tbh i tested on only 1 of 3 packs, maybe there was some kind of json bug or so

**Comment by hedgehog1029** [comment]
I couldn't immediately reproduce it locally. Seems to be some kind of issue with the manifest? I would've thought the parser would throw before it got to that point though.

**Comment by Bartixxx32** [comment]
yeah, im still working on it, now in modpack creator i have same error and will try use same workspace, but older (working) commit revision before gradle update

**Comment by Bartixxx32** [comment]
And, on commit 8e1337d603c4c66501b0a7bb34e31c154e9c7d86 is working

Gonna diff the _upload folder between new and old creator, but i guess that not the point

**Comment by Bartixxx32** [comment]
If you want, i can share with my workspace to you, whih is not working on latest commit for some reason

**Comment by Bartixxx32** [comment]
I think i found an issue, 
![image](https://user-images.githubusercontent.com/16412925/165414755-2a4e87ea-35bf-4a80-b86a-3d4a2be3a330.png)

**Comment by hedgehog1029** [comment]
Oh yeah that's definitely a problem. I guess 1.7.10 doesn't really *have* jvm arguments? Hang on, I have a test pack for that locally...

**Comment by hedgehog1029** [comment]
OK yup I can reproduce this with a 1.7.10 pack locally

**Comment by Bartixxx32** [comment]
Thats nice to hear!

**Comment by Bartixxx32** [comment]
Is 0a5abee9561df3c9b278c82ade2edaec329a6b0b with fixed issue i mentiones? or not yet?

**Comment by Bartixxx32** [comment]
closed by mistake :P

**Comment by hedgehog1029** [comment]
Should be fixed in 2fe96e443567dbf503d060d480e842c7aed2e6fb :)

**Comment by Bartixxx32** [comment]
Thanks a lot gonna test for sure!
